client/src/components/feedbacks/feedback.js

```diff
@@ -0,0 +1,83 @@
+import React, { Component } from 'react';
+import { Container } from 'react-bootstrap';
+import axios from 'axios';
+import FeedbackSend from './send';
+import FeedbackComment from './comment';
+import '../../style/feedback.css';
+
+const id = require('shortid');
+
+class Feedback extends Component {
+  constructor() {
+    super();
+
+    this.state = {
+      allFedback: [],
+      pieceFedback: [],
+      maxCommentShow: 1,
+      start: 0,
+      last: 0,
+    };
+  }
+
+  componentDidMount() {
+    axios.get('http://localhost:8080/api/feedback/2')
+      .then((response) => {
+        this.setState((prevState) => {
+          return {
+            allFedback: response.data,
+            pieceFedback: response.data.slice(prevState.last, prevState.maxCommentShow),
+            last: prevState.maxCommentShow,
+          };
+        });
+      });
+  }
+
+  onClickShowMoreComment() {
+    this.setState((prevState) => {
+      return {
+        pieceFedback: prevState.allFedback.slice(prevState.start, prevState.last + prevState.maxCommentShow),
```

Comment:
You can use spellchecker in your ide to avoid type mistakes `pieceFedback` https://github.com/streetsidesoftware/vscode-spell-checker